.github/workflows/test.yml

```diff
@@ -0,0 +1,80 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+
+    tags:
+
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  WEB3_PUBLIC_KEY: '0x5fbb58c8b15E498442b72E2c8Dad82266ea7e135'
+  SERVICE_SOLVER: "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC"
+  SERVICE_MEDIATORS: "0x90F79bf6EB2c4f870365E785982E1f101E93b906"
+  WEB3_PRIVATE_KEY: 'beb00ab9be22a34a9c940c27d1d6bfe59db9ab9de4930c968b16724907591b3f'
```

Comment:
Is this private key ok to expose here?

Reply:
Well it should be migrated to repo secrets prior to PR merge.